### PR TITLE
Resurrect EIP-1328

### PR DESCRIPTION
**Pull request by ligi**
EIP-1328 is heavily used in WalletConnect - and very unlikely to change - but as I understand the process we now need to move it to Draft then to Review then to Last Call then to Final. Starting this process now - cc @pedrouid

### CODE REVIEW

**Comment by pedrouid** [comment]
Seconded! We should also update the spec with parameters for v1.0 and v2.0

**Comment by axic** [comment]
I think you can bring it to `Review` directly given it is mostly ready.

**Comment by ligi** [comment]
@axic thanks - but according to this state-transition diagram I cannot:

image

**Review by axic — CHANGES_REQUESTED** [review]
Please update this to the current EIP-1 standards:
1. Add `description:` preamble  and remove `Simple Summary`
2. Optionally add `Motivation`
3. Remove `References`
4. Add `Backwards Compatibility`
5. Add `Security Considerations`
6. Make sure at least one author has a GitHub handle and not an email

**Comment by axic** [comment]
> @axic thanks - but according to this state-transition diagram I cannot:

Good catch. I think this is somewhat inconsistent, given if you have caught the bot at the time of making the change, you wouldn't need to do two steps.

**Comment by ligi** [comment]
I actually caught it the first time: https://github.com/ethereum/EIPs/pull/4020#issuecomment-922449227
Unfortunately I did not catch the second attempt ..-)

Do we really need to add empty sections for backward compatibility and security considerations when they do not apply?

**Comment by axic** [comment]
> Do we really need to add empty sections for backward compatibility and security considerations when they do not apply?

It seems so.

**Comment by MicahZoltu** [comment]
I believe we discussed this and agreed that if the author wants to go from stagnant to Review that is fine.

The Backward Compatibility and Security Consideration sections are required.  At the least they should have a one-line blurb that says "There are no known backward compatibility issues introduced by this EIP" or "There are no known security issues presented by the introduction of this EIP."

**Comment by lightclient** [comment]
Bumping as this is heavily used in WalletConnect and it would be great to move to final

**Comment by ligi** [comment]
I did most things @axic pointed out changed from EIP-1 in style - just left

* Add Backwards Compatibility
* Add Security Considerations

Also @pedrouid can you add the 2.0 changes you mentioned?

File: EIPS/eip-1328.md
Comment (ligi) [inline EIPS/eip-1328.md:15]: @pedrouid do we perhaps want to remove "This URI can then be shown either as a QR code or as a link." - think it adds no value and restricts us without a need. e.g. how does the "copy" fall in place there.

**Comment by pedrouid** [comment]
Thanks @ligi for resurrecting this EIP. I will look into this by Tuesday to include the missing sections and parameters for v2.0

**Comment by pedrouid** [comment]
Sorry for the delay.... I have created a PR to Ligi's PR to include the v2.0 parameters: https://github.com/ligi/EIPs/pull/2

File: EIPS/eip-1328.md
Comment (pedrouid) [reply EIPS/eip-1328.md:15]: I feel like this is already addressed when "link" is mentioned since it's a common pattern to include `copy to clipboard` for website links as well

**Comment by ligi** [comment]
@pedrouid thanks for the final changes for WC2.0!
@MicahZoltu thanks for your suggestions - unfortunately I saw them after merging @pedrouid 's and then got this when trying to merge yours:
image
So I changed it myself and gave you credits in the commit msg - sorry for not seeing it earlier

Think this one should be mergeable now.

**Comment by MicahZoltu** [comment]
@SamWilsn @lightclient @axic This has been sitting waiting for a review for quite a while.  Any chance one of you have some time to take a look?

File: EIPS/eip-1328.md
Comment (SamWilsn) [inline EIPS/eip-1328.md:3]:
```suggestion
title: WalletConnect URI Format
```

File: EIPS/eip-1328.md
Comment (SamWilsn) [inline EIPS/eip-1328.md:4]:
```suggestion
description: Define URIs for initiating connections between applications and wallets
```

File: EIPS/eip-1328.md
Comment (SamWilsn) [inline EIPS/eip-1328.md:57]: This "Rationale" section reads more like it should belong in the "Motivation" section.

"Motivation" explains why the EIP as a whole needs to exist, while "Rationale" should explain individual choices made in the EIP itself.

**Review by SamWilsn — CHANGES_REQUESTED** [review]

**Comment by SamWilsn** [comment]
Opened https://github.com/ligi/EIPs/pull/5 with my suggestions.

**Comment by ligi** [comment]
merged

**Review by SamWilsn — APPROVED** [review]
I think we're good to go!

**Comment by SamWilsn** [comment]
@ligi is this good to go to review?

**Comment by ligi** [comment]
yes!

**Review by Brandylee24 — APPROVED** [review]

**Pull request merged**
